app/src/main/java/org/mozilla/fenix/share/ShareFragment.kt

```diff
@@ -65,6 +65,10 @@ class ShareFragment : AppCompatDialogFragment(), CoroutineScope {
             }
         )
 
+        view.share_wrapper.setOnClickListener {
```

Comment:
I appreciate the commit. While there's no reason this isn't correct, it would be cleaner to make the change from the View layer. i.e. ```container.setOnClickListener { actionEmitter.onNext(ShareAction.Close) }``` in `ShareUIView.kt`.

This would technically make the closeButton listener redundant.

Reply:
Thanks for the feedback. PR updated